manifests/init.pp

```diff
@@ -334,6 +336,7 @@
   Boolean $tftp_managed = true,
   Boolean $tftp_manage_wget = true,
   Optional[Stdlib::Absolutepath] $tftp_root = $foreman_proxy::params::tftp_root,
+  Optional[Stdlib::Absolutepath] $system_image_root = $foreman_proxy::params::system_image_root,
```

Comment:
You never define it in `params.pp`.

Note that when there is a static value (meaning nothing derived from the OS or anything) we put it here. `tftp_root` is OS-dependent so it's "special".

Reply:
Done.